benchmark_runner/common/oc/oc.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, kubeadmin_password: str = ''):
         self.__kata_csv = self.__environment_variables_dict.get('kata_csv', '')
         self.__worker_disk_prefix = self.__environment_variables_dict.get('worker_disk_prefix', '')
         self.__worker_disk_ids = self.__environment_variables_dict.get('worker_disk_ids', '')
+        self.__exclude_default_operators = self.__environment_variables_dict.get('exclude_default_operators', '')
+        self.__exclude_operators_pattern = "|".join(self.__exclude_default_operators)
```

Comment:
`self.__exclude_default_operators` needs to be parsed to convert it into a Python array.

Reply:
Its a list and then I parse it "|" for exclude it in python, here:
`upgrade_versions = self.run(f"{self._cli} get csv -n {namespace} -o json | jq -r '.items[] | select(.metadata.name | test(\"{self.__exclude_operators_pattern}\") | not) | .spec.version'".splitlines()`

Reply:
Right, but IIRC environment_variables_dict.get() does not parse whatever it gets to create an array; it just returns a string.  That string needs to be parsed into an array for join() to work.